brianlong/www.validators.app
# 186628351 redesign header and footer

#### What's this PR do?
- As in title

#### How should this be manually tested?
- Run the server and review in browser
- Check also mobile versions 

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186628351)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
